### PR TITLE
LPD-44634 Clean inline styles to support style-src '[NONCE]' directive

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[LPD-44634](https://liferay.atlassian.net/browse/LPD-44634)

## Changes

Remove inline styles, so customers can define Content Security Policies with a restrictive directive.
In general, inline (those included inside an HTML tag) style attributes are discouraged when implementing CSPs → [CSP: style-src - HTTP | MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src#unsafe_inline_styles) so the goal of this epic is to extract all inline styles ingrained in the modiules' code to their own CSS files or, if that’s not possible, to an <aui:style> tag at the very least, so that it contains a CSP nonce attribute when rendered.

[Documentation](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/3222143001/LPD-18227+-+Ensure+the+support+of+CSP+directives+script-src+script-src-attr+style-src#Implementation)

[LPD-44634]: https://liferay.atlassian.net/browse/LPD-44634?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/color_picker_input.jsp [hunk 1]
```diff
@@ -23,7 +31,7 @@ String name = ParamUtil.getString(request, "name");
 			<div class="clay-color input-group">
 				<div class="input-group-item input-group-item-shrink input-group-prepend">
 					<div class="input-group-text">
-						<button class="btn clay-color-btn dropdown-toggle" style="border-width: 0px; height: 28px; width: 28px;" title="<%= color %>" type="button" />
+						<button class="btn clay-color-btn color-button-style dropdown-toggle" title="<%= color %>" type="button" />
```
Comment (victorg1991) [inline modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/color_picker_input.jsp:34]: `color-picker-color-button` maybe? What do you think?

Comment (victorg1991) [reply modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/color_picker_input.jsp:34]: Also, can't we remove this style and use the css class `clay-color-btn-bordered`? 

It seems that is the one used by the react component and here it is missing

Comment (BarbaraCabrera) [reply modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/color_picker_input.jsp:34]: image

Thanks

**Comment by victorg1991** [comment]
Niiice :clap:

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**